### ISSUE TITLE
Automatically Cleanup Backups

### ISSUE THREAD

**Issue by InvisaMage** [standardnotes/forum#495]
The backup system is great but I've got backups that are over a year old now and are no longer worth keeping. It would be nice if there was a way to delete old backups within the user interface that are a few months old. I know I could delete them manually but that's not as intuitive.

**Comment by mobitar** [comment]
Yeah, possibly. But I'd say this falls into the "luxury" category, and we're not quite there yet.
